style.css

```diff
@@ -1,7 +1,58 @@
+body {
+  display: flex;
+  flex-direction: column;
+  max-width: 320px;
+  background-image: url(images/image_geometry_1.png);
+  background-repeat: no-repeat;
+}
```

Comment:
[OPTIONAL] Kindly make sure that your project is responsive until desktop size. You could try removing the max-width property in this block of code. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/199642943-5cc7a280-f6ff-4db3-b678-e56ee4deea02.png)

Reply:
Thaks for the sugestion